lldb/test/API/tools/lldb-dap/attach/TestDAP_attach.py

```diff
@@ -227,3 +228,46 @@ def test_terminate_commands(self):
             pattern=terminateCommands[0],
         )
         self.verify_commands("terminateCommands", output, terminateCommands)
+
+    def test_session_id_update(self):
+        program = self.build_and_create_debug_adapter_for_attach()
+        self.process = subprocess.Popen(
+            [program],
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+
+        postRunCommands = [
```

Comment:
for the sake of readability, can you add a comment here about the env var VSCODE_DEBUG_SESSION_ID and how you are using it here?